packages/url_launcher/url_launcher/README.md

```diff
@@ -53,6 +53,28 @@ Common schemes supported by both iOS and Android:
 
 More details can be found here for [iOS](https://developer.apple.com/library/content/featuredarticles/iPhoneURLScheme_Reference/Introduction/Introduction.html) and [Android](https://developer.android.com/guide/components/intents-common.html)
 
+### Encoding URLs
+
+URLs should be safely encoded, espeically when including spaces or other special characters. We can do this using dart's `Uri` helper methods included in `dart:core`:
```

Comment:
Nit: the rest of this README appears to use "we" to mean the plugin maintainers, but this is a step that would be done by clients of the plugin, so it would be clearer to say "This can be done using" rather than "We can do this using".